lib/pages/chat_blank/chat_blank.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:fluffychat/pages/chat_blank/chat_blank_style.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:fluffychat/utils/string_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+
+class ChatBlank extends StatelessWidget {
+  final bool loading;
+  const ChatBlank({this.loading = false, Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Add invisible appbar to make status bar on Android tablets bright.
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        elevation: 0,
+        backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      ),
+      extendBodyBehindAppBar: true,
+      body: loading
+          ? ChatBlankLoading(context: context)
+          : ChatBlankNotChat(context: context),
+    );
+  }
+}
+
+class ChatBlankNotChat extends StatelessWidget {
```

Comment:
```suggestion
class _ChatBlankNotChat extends StatelessWidget {
```
private class otherwise create new file for one class